lib/multigrid.in.hpp

```diff
@@ -32,29 +32,59 @@ namespace quda
   template <class F> auto create_color_spinor_copy(cvector_ref<F> &fs, int nVec, QudaFieldOrder order)
```

Comment:
Can we have some documentation on what `create_color_spinor_copy` and `expand_color_spinor` does? When we come back at some point they would be useful.

Reply:
d123db48dcca82ceaf57447fe0d7594ff56e49d8